server/src/main/java/org/elasticsearch/rest/action/cat/RestShardsAction.java

```diff
@@ -237,6 +237,7 @@ protected Table getTableWithHeader(final RestRequest request) {
 
         table.addCell("path.data", "alias:pd,dataPath;default:false;text-align:right;desc:shard data path");
         table.addCell("path.state", "alias:ps,statsPath;default:false;text-align:right;desc:shard state path");
+        table.addCell("refresh.is_search_idle", "alias:rsi,refreshSearchIdle;default:false;text-align:right;desc:shard idle state");
```

Comment:
I wasn't really sure what to name this/where to put it relative to the other stats. `refresh.*` seemed like the best field name, but unlike all of the other `refresh.*` stats, this value comes from `ShardStats` so in the `buildTable(...)` method below it seemed like it fit better with the other `ShardStats`. But it's a bit of a misfit in both sections so I'd appreciate thoughts on this.